src/ansys/fluent/core/filereader/case_file.py

```diff
@@ -567,7 +567,7 @@ class CaseFile(RPVarProcessor):
 
     Methods
     -------
-    get_mesh
+    get_mesh()
```

Comment:
`get_mesh()` actually returns an object of type `Mesh`, so the docstring can be improved here. We can also add a type hint for the return type.

Reply:
listed in documentation [super-issue](https://github.com/ansys/pyfluent/issues/3456).